### ISSUE TITLE
Confirmed Count Anomaly - Japan - 01-22-2020, 01-23-2020

### ISSUE THREAD

**Issue by MelbourneDeveloper** [CSSEGISandData/COVID-19#2114]
On [line 37](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/01-22-2020.csv#L37) of 01-22-2020 the confirmed count in Japan is 2.

On [line 37]https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/01-23-2020.csv#L37) of 01-23-2020.csv the confirmed count in Japan is 1.

The account actually decremented by one. This means that it is impossible to calculate an accurate running total for Japan.

**Comment by MelbourneDeveloper** [comment]
Another case for Washington [line 20] (https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/03-17-2020.csv#L20) of 03-17-2020.csv the confirmed count is 1076

Another case for Washington [line 20] (https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/03-18-2020.csv#L22) of 03-18-2020.csv the confirmed count is 1014

**Comment by blisx264** [comment]
30th March NSW 1918 
7th April Norway 1160

More Australian values can be found here -> https://covid-19-au.com/

**Comment by MelbourneDeveloper** [comment]
Yamhill Oregon

26 [2020 6 4](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/04-07-2020.csv#L2570)

25 [2020 7 4](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/04-07-2020.csv#L2570)

**Comment by MelbourneDeveloper** [comment]
I can write a log of where these issues appear. Tell me how you would like it formatted and I can post it.

**Comment by lexchis** [comment]
I can't seem to find an answer to this: are the counts in the daily report files (mm-dd-yyyy.csv) cumulative totals, or newly recorded as of that date?

**Comment by MelbourneDeveloper** [comment]
@lexchis thanks for looking into it. Nearly all the figures are cumulative totals. That's the only sensible way to interpret the data unless I've missed something.

If you take any row, Deaths + Recovered + Active = Confirmed. For example:

State | Region | Date | Lat | Long | Confirmed | Deaths | Recovered | Active | --
Victoria | Australia | 2020-04-11 22:52:46 | -37.8136 | 144.9631 | 1265 | 14 | 926 | 325 | Victoria, Australia

It could be that the intention of the value was to represent an adjustment of some description but there are no notes to indicate this. I think that if these values do represent anadjustment, a column should be added to point to a reason why it was adjusted.

It is not possible that they could be "newly recorded as of that date" because many numbers are massive and it's not possible that they could represent new case figures for a single day. For example:

| Italy | 2020-04-11 22:45:13 | 41.87194 | 12.56738 | 152271 | 19468 | 32534 | 100269 | Italy
| Region | Date | Lat | Long | Confirmed | Deaths | Recovered | Active | --

**Comment by MelbourneDeveloper** [comment]
@lexchis when you find out the answer, could you please make sure this is documented in the readme section? I think it's really important that the meaning of each column is very clear.

**Comment by lexchis** [comment]
I believe you are right. There was at one time a big jump in the numbers 
as of around Mar 21, which didn't seem plausible even given exponential 
growth - it rather seemed explicable only by a change from daily new 
cases to cumulative cases. Those files appear to have been revised now 
so the growth in case numbers is more nearly continuous.

I have also found some of the .csv files which were unreadable. Some had 
non-ascii characters in the first 3 bytes. Others (4-09 to 4-11) simply 
crashed the Ruby CSV iterator for reasons I haven't been able to 
ascertain. I fixed that by running a grep on the .csv file to extract 
only those states of interest to me.

Thanks and stay well, Lex

**Comment by MelbourneDeveloper** [comment]
@lexchis 
> Those files appear to have been revised now so the growth in case numbers is more nearly continuous.

Yes, but the anomaly is still there which makes throws off any calculations if you include the earlier data

> I have also found some of the .csv files which were unreadable. Some had
non-ascii characters in the first 3 bytes. Others (4-09 to 4-11) simply
crashed the Ruby CSV iterator for reasons

I've had no issues with C#.

**Comment by MelbourneDeveloper** [comment]
This is a [related problem](https://github.com/CSSEGISandData/COVID-19/issues/2114)

**Comment by MelbourneDeveloper** [comment]
This is a comprehensive list:
https://github.com/CSSEGISandData/COVID-19/issues/2256
